app/views/line_items/_line_item_fields.html.erb

```diff
@@ -15,8 +15,9 @@
           <span class="li-name w-100">
             <%= field.input :item_id,
                             disabled: requested.present?,
-                            collection: @items, prompt: "Choose an item",
-                            include_blank: "",
+                            collection: @items,
+                            prompt: "Choose an item",
+                            include_blank: false, # We've got a prompt, so no need for an include blank
```

Comment:
I can remove this comment later, but I observed the item choice dropdown was showing some weird UI behavior, screenshot below of what I was seeing

<img width="1245" alt="image" src="https://github.com/user-attachments/assets/96260db8-5a41-4763-898c-d02626923bcd">

Reply:
With the change, here is what it looks like:
<img width="1255" alt="image" src="https://github.com/user-attachments/assets/3a427fcc-3c9a-402f-a416-11c2e2e7661c">